keyboards/handwired/neodox/config.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 MatteoDM <matteodalmo@gmail.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Handness */
+#define SPLIT_HAND_MATRIX_GRID GP5, GP15    /* row first because the board is col2row */
+
+/* Disable diode intersection in matrix data */
+#define MATRIX_MASKED
```

Comment:
IIRC, this shouldn't be needed anymore with the split hand matrix grid config, as this should get generated automatically, if the settings are in the info.json.